pos_no_bank_statement/model/point_of_sale.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# This file is part of OpenERP. The COPYRIGHT file at the top level of
+# this module contains the full copyright notices and license terms.
+
+from openerp import models, api
+
+
+class POSOrder(models.Model):
+    _inherit = "pos.order"
+
+    @api.v7
```

Comment:
@pedrobaeza, thanks but I can not make it work. With your code I get `TypeError: add_payment() takes exactly 2 arguments (6 given)` error. Are you sure Odoo knows how to map inherited object written in v7 to v8?

Cheers
Andrius